packages/babel-plugin-transform-decorators-2/test/fixtures/transformations/method-and-class-decorators/expected.js

```diff
@@ -0,0 +1,6 @@
+let Foo = babelHelpers.decorate(class Foo {
+  method() {}
+
+  undecorated() {}
+
+}, [["undecorated"]], [["method", [methDec]]], void 0)([classDec]);
```

Comment:
(Having not looked at the decorate implementation yet) Too many arrays? Some of these look like they could be flattened one level, but I'm only guessing at the meaning of each parameter.